### PR TITLE
Rebalance

### PR DESCRIPTION
**Pull request by h00dy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+From Vagrant box:
+```sh
+ubuntu:~ sudo su - alamo
+alamo@ubuntu-xenial:~$ cd /srv/alamo_scheduler_ex/
+alamo@ubuntu-xenial:/srv/alamo_scheduler_ex$ REPLACE_OS_VARS=true RELEASE_CONFIG_DIR=/etc/alamo bin/alamo remote_console
+
+```
+While in remote shell, run:
+```elixir
+iex(alamo@192.168.99.2)2> JobStore.Database.add_node(:"alamo@192.168.99.3")
```
Comment (benethor) [inline README.md]: would be nice to describe what 192.*.3 means - currently looks like a typo

File: README.md [hunk 2]
```diff
@@ -1,4 +1,86 @@
...
+* `frequency` - defines check intervals in seconds
+* `timestamp` - event creation time in Unix timestamp
+* `triggers` - list of check triggers, empty list means remove check from scheduler
+* `uuid` = unique check identifier, used as a primary key in scheduler's JobStore
+.
+
+You can see if job was added by entering status page at: http://192.168.99.2:8080/
+or looking for job by checking http://192.168.99.2:8080/api/some_uuid123 (assuming that check has uuid='some_uuid123')
+
+**Removing job from scheduler**
```
Comment (benethor) [inline README.md]: a job/jobs

File: apps/scheduler/test/zmq_test.exs [hunk 3]
```diff
@@ -15,26 +16,26 @@ defmodule SchedulerZMQTest do
...
-    zmq_envs = Application.get_env(:scheduler, :zmq)
-    {:ok, pull_socket} = :chumak.socket(:pull)
-    {:ok, connect_pid_prod} = :chumak.connect(pull_socket, :tcp, @ip, zmq_envs.prod)
-    :sys.get_state(connect_pid_prod)
-    {:ok, connect_pid_test} = :chumak.connect(pull_socket, :tcp, @ip, zmq_envs.test)
-    :sys.get_state(connect_pid_test)
-    {:ok, connect_pid_dev} = :chumak.connect(pull_socket, :tcp, @ip, zmq_envs.dev)
-    :sys.get_state(connect_pid_dev)
-    {:ok, pull_socket: pull_socket, check_payload: check_payload}
+    {:ok, ctx} = :czmq.start_link()
```
Comment (benethor) [inline apps/scheduler/test/zmq_test.exs]: wouldn't connect() be more intuitive?

Comment (h00dy) [reply apps/scheduler/test/zmq_test.exs]: :czmq is an external library

File: README.md [hunk 4]
```diff
@@ -1,4 +1,87 @@
...
+From Vagrant box:
+```sh
+ubuntu:~ sudo su - alamo
+alamo@ubuntu-xenial:~$ cd /srv/alamo_scheduler_ex/
+alamo@ubuntu-xenial:/srv/alamo_scheduler_ex$ REPLACE_OS_VARS=true RELEASE_CONFIG_DIR=/etc/alamo bin/alamo remote_console
+
+```
+While in remote shell, run:
+```elixir
+iex(alamo@192.168.99.2)2> JobStore.Database.add_node(:"alamo@192.168.99.3")
```
Comment (h00dy) [inline README.md]: fixed

**Pull request merged**